community-operators/etcd/etcdoperator.v0.9.4.clusterserviceversion.yaml

```diff
@@ -0,0 +1,353 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: etcdoperator.v0.9.4
+  namespace: placeholder
+  annotations:
```

Comment:
Additional fields in annotations are required for this CSV to be rendered in the UI (outlined here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md).